src/VirtoCommerce.CoreModule.Data/Currency/CurrencyService.cs

```diff
@@ -17,12 +17,14 @@ public class CurrencyService : ICurrencyService
         private readonly Func<ICoreRepository> _repositoryFactory;
         private readonly IEventPublisher _eventPublisher;
         private readonly IPlatformMemoryCache _platformMemoryCache;
+        private readonly IMoneyRoundingPolicy _moneyRoundingPolicy;
 
-        public CurrencyService(Func<ICoreRepository> repositoryFactory, IEventPublisher eventPublisher, IPlatformMemoryCache platformMemoryCache)
+        public CurrencyService(Func<ICoreRepository> repositoryFactory, IEventPublisher eventPublisher, IPlatformMemoryCache platformMemoryCache, IMoneyRoundingPolicy moneyRoundingPolicy)
         {
             _repositoryFactory = repositoryFactory;
             _eventPublisher = eventPublisher;
             _platformMemoryCache = platformMemoryCache;
+            _moneyRoundingPolicy = moneyRoundingPolicy;
```

Comment:
Missed  `_moneyRoundingPolicy` initialization  to returned currencies from the service